crate_universe/private/crates_vendor.bzl

```
@@ -129,6 +129,10 @@ def _write_config_file(ctx):
 
     output_pkg = _get_output_package(ctx)
 
+    workspace_name = ""
```

Minor nit: Could this instead be
```suggestion
    workspace_name = ctx.workspace_name
    if workspace_name == "__main__":
        workspace_name = ""
```

---

Done.